src/store/middlewares/account.js

```diff
@@ -239,6 +247,11 @@ const accountMiddleware = store => next => (action) => {
     case actionTypes.updateTransactions:
       votePlaced(store, action);
       break;
+    case actionTypes.accountUpdated:
```

Comment:
If this case is empty then it should be removed to it goes to `default: break;`

Reply:
This case is not empty. I have combined two cases together because the function to use in both the cases is the same.